enterprise/server/test/integration/remote_cache/remote_cache_test.go

```diff
@@ -239,7 +239,7 @@ func TestBuild_RemoteCacheFlags_Compression_SecondBuildIsCached(t *testing.T) {
 
 func TestBuild_RemoteCache_ScoreCard(t *testing.T) {
 	app := buildbuddy_enterprise.RunWithConfig(
-		t, buildbuddy_enterprise.DefaultAppConfig(t), buildbuddy_enterprise.NoAuthConfig,
+		t, buildbuddy_enterprise.DefaultAppConfig(t), buildbuddy_enterprise.NoAuthConfigRunfilePath,
```

Comment:
imo we should keep these callsites the same, since `RunfilePath` seems like more of an implementation detail of the package.

i.e. change `NoAuthConfigRunfilePath` => `NoAuthConfig`, and change the current `NoAuthConfigName` => `noAuthConfigName` (make it private). (Same for the default config)